src/main/webapp/index.html

```diff
@@ -26,69 +26,67 @@ <h1>Census Viz</h1>
+            <option data-value="work" value="work in"></option>
+            <option data-value="moved" value="moved to"></option>
+          </datalist>
           </input>
 
-          <input type="list" list="location" name="location" id="location-list"
-              autocomplete="off" required onkeyup="validateInput('location')"
-              onfocus="storeValueAndEmpty('location')"
-              onfocusout="replaceValueIfEmpty('location')"
-              value="each U.S. state">
-            <datalist id="location"></datalist>
+          <input type="list" list="location" name="location" id="location-list" autocomplete="off" required
+            onkeyup="validateInput('location')" onfocus="storeValueAndEmpty('location')"
+            onfocusout="replaceValueIfEmpty('location')" value="each U.S. state">
+          <datalist id="location"></datalist>
           </input>
           in
-          <input type="list" list="year" name="year" id="year-list"
-              autocomplete="off" required onkeyup="validateInput('year')"
-              onfocus="storeValueAndEmpty('year')"
-              onfocusout="replaceValueIfEmpty('year')"
-              value="2018" pattern="201[0-8]">
-            <datalist id="year">
-              <option value="2010"></option>
-              <option value="2011"></option>
-              <option value="2012"></option>
-              <option value="2013"></option>
-              <option value="2014"></option>
-              <option value="2015"></option>
-              <option value="2016"></option>
-              <option value="2017"></option>
-              <option value="2018"></option>
-            </datalist>
+          <input type="list" list="year" name="year" id="year-list" autocomplete="off" required
+            onkeyup="validateInput('year')" onfocus="storeValueAndEmpty('year')"
+            onfocusout="replaceValueIfEmpty('year')" value="2018" pattern="201[0-8]">
+          <datalist id="year">
+            <option value="2010"></option>
+            <option value="2011"></option>
+            <option value="2012"></option>
+            <option value="2013"></option>
+            <option value="2014"></option>
+            <option value="2015"></option>
+            <option value="2016"></option>
+            <option value="2017"></option>
+            <option value="2018"></option>
+          </datalist>
           </input>
           ?
         </p>
         <input type="submit" value="go">
       </form>
     </div>
     <div id="result">
+      <div id="map-options">
+        <div class="map-option">Flat map</div>
+        <div class="map-option">
+          <label class="switch" id="map-toggle-switch">
+            <input type="checkbox" id="map-toggle" onchange="toggleMap()">
+            <span class="slider round"></span>
+          </label>
+        </div>
+        <div class="map-option">Map overlay</div>
+      </div>
       <div id="map-title"></div>
```

Comment:
Could we add a line break here potentially, or alternatively put some padding around the toggle using CSS? The toggle, the title, and the map are really crowded together, and I think it should be more clear that the title goes with the map. (That will also let people screenshot those two parts alone more easily if they want to, without getting the toggle in the way -- I'm always annoyed when a site makes it hard for me to do that.)

Reply:
Oh, this will be even more important when colors and map toggle are merged. I'm doing that now for my PR and it's getting kind of messy up there lol.

Reply:
Good point, added some space between toggle switch and map title